build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -71,9 +71,12 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                 }
                 else
                 {
-                    // This is an old drop of latest so remove all old files to ensure a clean state
+                    Regex versionFileRegex = new Regex(@"(?<version>\d\.\d\.\d)-(?<release>.*)?");
```

Comment:
We should really consider splitting the "channels" between CLI and core-setup. But maybe that is what @sokket is doing right now, right?